src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1015,7 +1019,9 @@ def __getattr__(self, name: str):
         except KeyError:
             raise AttributeError(
                 allowed_name_error_message(
-                    "Settings objects", name, self.get_active_child_names()
+                    context="Settings objects",
```

Comment:
Does this generate f"{trial_name} is not an allowed Settings objects name"?

Reply:
yes